wazuh/wazuh-kibana-app
# Fix broke format in discover after manual refresh of the index pattern

Hi team this PR fix an error that broked the format of Kibana's Discover after a manual refresh of  the index pattern. It happened due to we used to removed all metafields from the index pattern and the one that is in charge of the format is `_source`. Because an error from Kibana I have also added `_index` to the index pattern.

### How to test
- Go to Stack management > Advance settings and look for Meta fields setting, then change it to default and remove the current index pattern
- After this pass the health check and look for that setting again, there should only be `_source` and `_index`.
- If you change that setting and add another metafield and then pass the health check the setting should remain the same as you have previously setted

Closes #3215
